content/docs/start/experiments/visualization.md

```diff
@@ -0,0 +1,107 @@
+---
+title: 'Get Started: Visualization with Plots'
+---
+
+# Get Started: Visualization with Plots
+
+In this section, we'll add visualization to the [`example-dvc-experiments`][ede]
```

Comment:
Let's just make sure this matches whatever results from [visualization](https://github.com/iterative/dvc.org/pull/3050) please. And or merge that PR first (and then `master` into here + move the file into this location). Thanks

Reply:
Merged #3050. Will merge the most recent version here.

Reply:
This one now contains visualization from master.